README.md

```diff
@@ -33,6 +76,7 @@ jobs:
 * `account` - this is the account configured for the Conjur instance during deployment.
 * `host_id` - this is the Host ID granted to your application by Conjur when created via policy. e.g. `host/db/github_action`
 * `api_key` - this is the API key associated with your Host ID declared previously.
+* `authn_id` - this is the ID of Authn-JWT at Conjur
```

Comment:
See comment added on the header `### Required` above.

Reply:
Sure. Updated.